openfecwebapp/templates/legal-doc-search-results.html

```diff
@@ -0,0 +1,64 @@
+{% extends "layouts/main.html" %}
+{% import 'macros/legal-search.html' as legal_search %}
+{% set document_type_display_name = 'Advisory Opinions' %}
+
+{% block title %}
+Searching {{ document_type_display_name }} for "{{ query }}"
+{% endblock %}
+
+{% block body %}
+<header class="page-header slab slab--primary">
+  <span class="page-header__title">Legal resources // {{ document_type_display_name }}</span>
```

Comment:
Cool, I'll address this in a [separate PR](https://github.com/18F/openFEC-web-app/issues/1204).